app/components/user_list/index.tsx

```diff
@@ -175,8 +178,8 @@ export default function UserList({
                 isMyUser={currentUserId === item.id}
                 onPress={handleSelectProfile}
                 onLongPress={openUserProfile}
-                disabled={!canAdd}
-                selectable={true}
+                disabled={disabled}
```

Comment:
I forgot to change this on the UserList refactor, it works better now. The selector CompassIcon were being showed when it was not multiselect, that's now fixed.